clarity/enhancer/dnn/mc_conv_tasnet.py

```diff
@@ -316,13 +318,13 @@ def __init__(
         norm_type="gLN",
         causal=False,
     ):
-        super(TemporalBlock, self).__init__()
+        super().__init__()
         # [M, B, K] -> [M, H, K]
         conv1x1 = nn.Conv1d(in_channels, out_channels, 1, bias=False)
         prelu = nn.PReLU()
         norm = chose_norm(norm_type, out_channels)
         # [M, H, K] -> [M, B, K]
-        dsconv = DepthwiseSeparableConv(
+        dsconv = DepthwiseSeparableConv(  # pylint: disable=W1114
```

Comment:
This might be the intended behvaiour but [W1114](https://vald-phoenix.github.io/pylint-errors/plerr/errors/typecheck/W1114.html) seems important as otherwise the `out_channels` and `in_channels` are passed to `DepthwiseSeparableConv() in the wrong order ([PyLint issue defining the error](https://github.com/PyCQA/pylint/issues/2975)).

That may be intentional here and the error has been disabled.

Reply:
Yes, switch is actually needed. ie an output being passed to an input and vice versa, but triggers the false +ve pylint error. I thought best to add the specific disable so that the error doesn't persist and make people think it's wrong later.

Reply:
Cool, thought I'd check, thanks for the confirmation.